Dockerfile

```diff
@@ -1,9 +1,19 @@
-FROM golang:1.11-alpine
+FROM golang:1.11-alpine as build
 
 RUN apk add --no-cache tzdata
 
 WORKDIR /go/src/github.com/vshn/k8up
 COPY . .
 RUN go install -v ./...
 
-ENTRYPOINT [ "operator" ]
+# runtime image
+FROM docker.io/alpine:3
+WORKDIR /app
+
+RUN apk --no-cache add ca-certificates tzdata
```

Comment:
When using `scratch` these files could simply be copied from the build container.